R/print.data.table.R

```diff
@@ -64,15 +65,29 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
     }
     return(invisible(x))
   }
+  index_dt = NULL
```

Comment:
I wouldn't do this -- R lazy eval means we don't need to instantiate a variable in unvisited branches, so IMO it's just a distraction